client/src/components/Messaging/Message.js

```diff
@@ -42,6 +42,11 @@ export const Message = ({
         <div className={`${styles.messageContainer} ${!darkMode && styles.lightModeContainer}`}>
           {image && <Image src={image} maxWidth="300px" maxHeight="300px" />}
           {body}
+          {id !== undefined ? (
```

Comment:
I understand why you took the ID but but please add an additional key called `timestamp` to prevent future bugs/confusion. 
It's not expected that changing the `id` breaks `timestamp`.

Reply:
Got it!